### PR TITLE
Add default training parameters to classification refrence README

### PR DESCRIPTION
**Pull request by pmeier**
Addresses https://github.com/pytorch/vision/issues/1986#issuecomment-601351866.

### CODE REVIEW

File: references/classification/README.md [hunk 1]
```diff
@@ -4,7 +4,30 @@ This folder contains reference training scripts for image classification.
...
+### AlexNet and VGG
+
+Since `AlexNet` and `VGG*` do not include batch normalization, the default
+initial learning rate `--lr 0.1` is to high.
+
+```
+python main.py --model $MODEL --lr 1e-2
+```
+
+Here `$MODEL` is one of `alexnet`, `vgg11`, `vgg11_bn`, `vgg13`, `vgg13_bn`,
```
Comment (fmassa) [inline references/classification/README.md]: the `_bn` variants were trained with larger learning rate I believe.

Comment (pmeier) [reply references/classification/README.md]: Dang. I wrote it in the text above and than ignored it.
